index.html

```diff
@@ -0,0 +1,316 @@
+                        away.
+                        <div class="new-line"></div> That was where Dr. Guth came in.
+                        <div class="new-line"></div> Under some circumstances, a glass of water can stay liquid as the temperature falls below 32 degrees, until it is disturbed, at which point it will rapidly freeze, releasing latent heat.
+                        <div class="new-line"></div> Similarly, the universe could “supercool” and stay in a unified state too long. In that case, space itself would become imbued with a mysterious latent energy.
+                        <div class="new-line"></div> Inserted into Einstein’s equations, the latent energy would act as a kind of antigravity, and the universe would blow itself up. Since it was space itself supplying the repulsive force, the more space was created, the harder
+                        it pushed apart.
+                        <div class="new-line"></div> What would become our observable universe mushroomed in size at least a trillion trillionfold — from a submicroscopic speck of primordial energy to the size of a grapefruit — in less than a cosmic eye-blink.
+                        <div class="new-line"></div> Almost as quickly, this pulse would subside, relaxing into ordinary particles and radiation. All of normal cosmic history was still ahead, resulting in today’s observable universe, a patch of sky and stars billions of light-years
+                        across. “It’s often said that there is no such thing as a free lunch,” Dr. Guth likes to say, “but the universe might be the ultimate free lunch.”
+                        <div class="new-line"></div> Make that free lunches. Most of the hundred or so models resulting from Dr. Guth’s original vision suggest that inflation, once started, is eternal. Even as our own universe settled down to a comfortable homey expansion,
+                        the rest of the cosmos will continue blowing up, spinning off other bubbles endlessly, a concept known as the multiverse.
+                        <div class="new-line"></div> So the future of the cosmos is perhaps bright and fecund, but do not bother asking about going any deeper into the past.
+                        <div class="new-line"></div> We might never know what happened before inflation, at the very beginning, because inflation erases everything that came before it. All the chaos and randomness of the primordial moment are swept away, forever out of our
+                        view.
+                        <div class="new-line"></div> “If you trace your cosmic roots,” said Abraham Loeb, a Harvard-Smithsonian astronomer who was not part of the team, “you wind up at inflation.”
+                        <div class="new-line"></div>
+                </div>
+
+            </section>
+        </article>
+
+
+
```

Comment:
### Implement missing sections 
The following sections have not been implemented on the clone page. I have highlighted them below for easier reference. 

#### Suggested newsletters section

**NOTE: Do you mind also include in the implementation, the `READ 615 Comments` and the preceding social links as well?**

![Suggested-newsletters-section](https://user-images.githubusercontent.com/22345961/112322162-b77e0c00-8cc1-11eb-9d75-5167dba00c0f.png)

#### More into space and astronomy

![more-inspace-and-astronomy](https://user-images.githubusercontent.com/22345961/112322206-c4026480-8cc1-11eb-8411-08b31cad9b74.png)

#### Editors pick

![editors-pick-footer](https://user-images.githubusercontent.com/22345961/112322262-ce246300-8cc1-11eb-800f-4f53fb028cd6.png)